application/controllers/SurveyAdministrationController.php

```
@@ -932,6 +932,26 @@ public function actionToggleQuickAction()
+     * This action is just for performance testing and should not be used in a production environment.
+     * It simply creates an amount of surveys.
+     *
+     * @return void
+     */
+    public function actionCreateTestSurveys() {
+        $service = new \LimeSurvey\Models\Services\BulkCreateSurveys(
+            Yii::app()->user->id,
+            'en',
+            1
+        );
+        $results = $service->run();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$results'. (undefined)

(UnusedLocalVariable)